### DISCUSSION TITLE
Azure authentication returns 502 response from Supabase server

### DISCUSSION THREAD

**Discussion by mbalslow** [supabase/supabase#5261] [Questions] [answered]
I have been using Azure authentication for months without any issues. Today, we are unable to login at all. Seems like something happened on the Supabase side of things?

After a successful login using the Azure authentication provider, for example:
`https://<supabase-id>.supabase.co/auth/v1/authorize?provider=azure&redirect_to=http://localhost:3000`

The redirect to localhost does not make a difference. Some error occurs if it is not present.

A error 502 response is returned from the Supabase server. Running version 1.29.4 of supabase-js but the same happens by simply running the flow manually using the url in a browser. 

![image](https://user-images.githubusercontent.com/49169557/151946004-d107bdd4-22f1-41c3-8051-c401b0d4f398.png)

![image](https://user-images.githubusercontent.com/49169557/151946117-00eee062-0f06-4c58-aa31-48260212fccc.png)

**Comment by GaryAustin1** [comment, marked as answer]
Might check this out:  
https://github.com/supabase/supabase/issues/4758

**Comment by J0** [comment]
Hey all,

Thanks for the query -- we haven't changed anything within the Azure provider logic.  This may have been a temporary outage but it's not possible to easily determine given the amount of time that has lapsed.

Going to mark as answered for now but let us know if you run into further errors.

Thanks!
